crates/ruff/src/rules/pycodestyle/rules/whitespace_around_keywords.rs

```diff
@@ -70,6 +80,8 @@ pub fn whitespace_around_keywords(line: &str) -> Vec<(usize, DiagnosticKind)> {
             diagnostics.push((after.start(), TabAfterKeyword.into()));
         } else if after.as_str().len() > 1 {
             diagnostics.push((after.start(), MultipleSpacesAfterKeyword.into()));
+        } else if after.as_str().is_empty() {
+            diagnostics.push((after.start(), MissingWhitespaceAfterKeyword.into()));
```

Comment:
Interestingly, `pycodestyle` does this a little differently -- can you help me understand how the implementation differs?

```
@register_check
def missing_whitespace_after_keyword(logical_line, tokens):
    r"""Keywords should be followed by whitespace.

    Okay: from foo import (bar, baz)
    E275: from foo import(bar, baz)
    E275: from importable.module import(bar, baz)
    E275: if(foo): bar
    """
    for tok0, tok1 in zip(tokens, tokens[1:]):
        # This must exclude the True/False/None singletons, which can
        # appear e.g. as "if x is None:", and async/await, which were
        # valid identifier names in old Python versions.
        if (tok0.end == tok1.start and
                keyword.iskeyword(tok0.string) and
                tok0.string not in SINGLETONS and
                tok0.string not in ('async', 'await') and
                not (tok0.string == 'except' and tok1.string == '*') and
                not (tok0.string == 'yield' and tok1.string == ')') and
                tok1.string not in ':\n'):
            yield tok0.end, "E275 missing whitespace after keyword"
```

Reply:
I think the current version we have here would flag `if None:`, right?

Reply:
@charliermarsh yes you're right, will try to improve this implementation.

Reply:
No prob! Most of the pycodestyle code thus far is nearly a 1:1 port of the original implementation, so you can probably check out their source for inspiration.